examples/getting_started/tf/README.md

```diff
@@ -159,11 +121,11 @@ python ./tf_fl_script_executor_cifar10.py \
        --fedprox_mu 1e-5 \
```

Comment:
we can remove this command since it is for the more advanced scaffold algorithm (and it covered under advanced/job_api/tf now)